classes/pickup/coin/coin_pickup.gd

```diff
@@ -21,10 +21,10 @@ func _ready_override() -> void:
 		vel.y = -7 * 0.53
 
 
-func _add_coins(num: int) -> void:
+func _add_coins(num: int, player) -> void:
```

Comment:
Consider adding a type hint to `player` here?

Reply:
Ahhh, yep, good call